persistence/persistence-mybatis/pom.xml

```diff
@@ -38,7 +38,7 @@
         <dependency>
             <groupId>com.h2database</groupId>
             <artifactId>h2</artifactId>
-            <version>1.4.192</version>
+            <version>1.4.197</version>
```

Comment:
I guess it should not hurt to update h2

Reply:
@sheridancbio yeah it's set to update in another PR already so it can either update now or later with the other PR. Doesn't really matter but to me sooner seems better